src/index.ts

```diff
@@ -1,36 +1,56 @@
-import {
-  WallyConnectorOptions,
-  RedirectOptions,
-} from './types';
+import { WallyConnectorOptions } from './types';
 
 import WallyConnector from './wally-connector';
 
-
 declare global {
   // eslint-disable-next-line no-var
-  var wally: WallyConnector
+  var wally: WallyConnector;
 }
 
+const checkInjected = () => {
+  if (!window.wally) {
```

Comment:
so do something like `window.WallyConnector.wally`? wouldn't that be pretty much the same thing? I know it's not perfect, but I don't think one global is _too_ bad.... we're going to need some sort of global to access the instance throughout the consuming application, and it's how other providers work, i.e. `window.ethereum` for metamask and gamestop.

Reply:
Yea, I think it's relatively minor but having too many globals can affect a developer's runtime so that's why it could be nice to namespace. `WallyConnector` is actually already global and so using the class name as the namespace could be a good place. I think the thing I'd caution on is that we should to be aware of what would interrupt the developer's environment and not strictly just ours since we're just building the SDK. They own the environment.

Reply:
was looking through the code again and we don't need the global anymore 🤷